src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1244,6 +1244,12 @@ public static class FontAndSizeSettings
             public static readonly string FontSizeWatermark = "Font Size Watermark";
             public static readonly string IncludeOriginData = "Include Origin Data";
          }
+
+         public static class ColorGrouping
+         {
+            public static string ColorGroupingDialogDescription = "Choose the metadata according to which the data will be grouped to the same color";
```

Comment:
First interation
"Select metadata used to group data by color:

Reply:
what about "Select metadata as criteria for color grouping" ?